src/ameba/spec/expect_issue.cr

```diff
@@ -47,6 +47,8 @@ require "./util"
 module Ameba::Spec::ExpectIssue
   include Spec::Util
 
+  class_property source : Source?
+
```

Comment:
this may not work well if someday we parallelize the tests

Reply:
What about returning `source` from `expect_issue`?

Reply:
yeah, this sounds good to me. Also, that will allow using `expect_correction` without `expect_issue` at all, right?

Reply:
That is correct (pun intended)